components/Topics/TopicsController.cs

```diff
@@ -534,10 +534,66 @@ public override IList<SearchDocument> GetModifiedSearchDocuments(ModuleInfo modu
 		}
 		#endregion
 
-		//Public Function ActiveForums_GetPostsForSearch(ByVal ModuleID As Integer) As ArrayList
-		//    Return CBO.FillCollection(DataProvider.Instance().ActiveForums_GetPostsForSearch(ModuleID), GetType(PostInfo))
-		//End Function
-	}
+        //Public Function ActiveForums_GetPostsForSearch(ByVal ModuleID As Integer) As ArrayList
+        //    Return CBO.FillCollection(DataProvider.Instance().ActiveForums_GetPostsForSearch(ModuleID), GetType(PostInfo))
+        //End Function
+        #region "IUpgradeable"
+        public string UpgradeModule(string Version)
+        {
+            switch (Version)
+            {
+                case "07.00.07":
+                    try
+                    {
+                        var fc = new ForumsConfig();
+                        fc.ArchiveOrphanedAttachments();
+                    }
+                    catch (Exception ex)
+                    {
+                        LogError(ex.Message, ex);
+                        Exceptions.LogException(ex);
+                        return "Failed";
+                    }
+
+                    break;
+                case "08.00.00":
```

Comment:
Hey @johnhenley ...  I don't think this will have any adverse impact, but it is referencing 8.0 code.  Do you know if there is anything underlying that we should be worried about before I process this?

Reply:
The 8.0 case needs to be removed because it's probably not going to compile. Let me know if you want me to do a pull request.

Reply:
Oh, then maybe the commit would be best?  I was about to try myself.

Reply:
You should be able to commit now as I resolved the conflicts. Not sure what happened but looks ok now